src/sql/workbench/parts/notebook/outputs/notebookMarkdown.ts

```diff
@@ -0,0 +1,240 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as path from 'path';
+
+import { URI } from 'vs/base/common/uri';
+
+import { IDisposable, dispose } from 'vs/base/common/lifecycle';
+import { RenderOptions } from 'vs/base/browser/htmlContentRenderer';
+import { IMarkdownString, removeMarkdownEscapes } from 'vs/base/common/htmlContent';
+import { IMarkdownRenderResult } from 'vs/editor/contrib/markdown/markdownRenderer';
+import marked = require('vs/base/common/marked/marked');
+import { defaultGenerator } from 'vs/base/common/idGenerator';
+import { revive } from 'vs/base/common/marshalling';
+import * as fs from 'fs';
+
+export class NotebookMarkdownRenderer {
```

Comment:
Can you leave a comment referencing which class we're basing this on? Periodically we might want to compare & see if we need to / should update anything.

Reply:
Ah I put a comment in renderMarkdown but not before the class declaration. I'll make sure there's a comment at the top too.